api/src/main/java/marquez/db/RunDao.java

```diff
@@ -192,6 +192,9 @@ LEFT OUTER JOIN (
       """)
   List<Run> findAll(String namespace, String jobName, int limit, int offset);
 
+  @SqlQuery("SELECT count(*) FROM runs_view AS r WHERE r.job_name = :jobName")
+  int countFor(String jobName);
```

Comment:
I think you need to add namespace here if I understand correctly to make the filter the same between the runs call and the count.

Reply:
Thanks, @phixMe . When I try adding the namespace to the filter query, the API returns an error saying that the runs_view lacks a namespace column. The logs also show that the endpoint already receives the correct namespace. So is this OK as-is, or is some other change required?

Reply:
You'd have to compare the runs query to the total count query... They need to be exactly the same except the count query should just have the `select count` at the beginning.